synapse/push/push_rule_evaluator.py

```diff
@@ -0,0 +1,361 @@
+# Copyright 2015, 2016 OpenMarket Ltd
```

Comment:
Is this file supposed to be here? I may have screwed up when trying to merge in develop

Reply:
That file was deleted.